app/scripts/templates/confirm_account_unlock.mustache

```diff
@@ -0,0 +1,22 @@
+<div id="main-content">
+  <header>
+    <h1 id="fxa-confirm-account-unlock-header">{{#t}}Unlock email sent{{/t}}</h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success">
+      {{#t}}Email resent{{/t}}
+    </div>
```

Comment:
> I'm not familiar with the CSS uses here - does this only become visible when the "resed email" action is invoked?

That's right. Unlike most other elements, both `.error` and `.success` are hidden by default. Visibility is [toggled](https://github.com/mozilla/fxa-content-server/blob/5cf7dfaba036d314e090deea0f56ee4aa73dbf1b/app/scripts/views/base.js#L42) in [views/base.js](https://github.com/mozilla/fxa-content-server/blob/5cf7dfaba036d314e090deea0f56ee4aa73dbf1b/app/scripts/views/base.js#L68).

Both `.error` and `.success` are usually empty by default, their content dynamically added after some action has completed/errored. In a few cases, only one success message is possible, so the text is placed inline in the HTML.